hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/requestdsl/StringMatcher.kt

```diff
@@ -37,4 +37,4 @@ class StringAssertion(
 @Suppress("ktlint:standard:function-naming")
 internal infix fun String.`is`(value: String) = StringAssertion(this, StringCondition.IS, value)
 internal infix fun String.isNot(value: String) = StringAssertion(this, StringCondition.IS_NOT, value)
-internal infix fun String.contains(value: String) = StringAssertion(this, StringCondition.CONTAINS, value)
+internal infix fun String.has(value: String) = StringAssertion(this, StringCondition.CONTAINS, value)
```

Comment:
Can't use contains because already exists on `String` - but this is just for the test data DSL anyway so can live with that.